tsdb_k8s_queries/challenges/default.json

```diff
@@ -326,4 +326,71 @@
         }{{ ", " if not loop.last else "" }}
         {%- endfor %}
       ]
-    }
+    },
+    {
+      "name": "storage",
+      "description": "A challenge centered just around storage. Indexes the two data sets and runs a force merge after that.",
+      "default": false,
+      "schedule": [
+        {
+          "name": "create-all-templates",
+          "operation": {
+            "operation-type": "create-composable-template",
+            "request-params": {
+              "create": "true"
+            }
+          }
+        },
```

Comment:
I think we are missing `delete-all-datastreams`, `delete-all-composable-template`, and `put-timestamp-template` tasks here, similarly to what is done in other challenges in this track. Challenge should restore a known good state before continuing with whatever it's supposed to do.